README.md

```diff
@@ -6,8 +6,9 @@
 [![Dependency Status](https://gemnasium.com/abonas/kubeclient.svg)](https://gemnasium.com/abonas/kubeclient)
 
 A Ruby client for Kubernetes REST api.
-The client supports GET, POST, PUT, DELETE on nodes, pods, secrets, services, replication controllers, namespaces, resource quotas, limit ranges, endpoints, persistent volumes, persistent volume claims, component statuses and service accounts.
+The client supports GET, POST, PUT, DELETE on all the entities available in kubernetes in both the core and group apis.
```

Comment:
Since the entities vs resources was not introduced here, I prefer to leave that out of this pr.
(currently the only mention of resources here is the Resource class)

I agree and would appreciate a fix for that. Maybe event alias all_entities with all_resources
(however before working on that I would consult with @abonas or @simon3z to see if there is a reason entities is used)